From fb5746826a0cfc1d5a8d9bbe98c01b194480a765 Mon Sep 17 00:00:00 2001
From: Antonio Borneo <borneo.antonio@gmail.com>
Date: Thu, 18 Jun 2020 02:08:18 +0200
Subject: [PATCH] usbip: tools: fix module name in man page
Git-commit: fb5746826a0cfc1d5a8d9bbe98c01b194480a765
Patch-mainline: v5.9-rc1
References: git-fixes

Commit 64e62426f40d ("staging: usbip: edit Kconfig and rename
CONFIG options") renamed the module usbip as usbip-host, but the
example in the man page still reports the old module name.

Fix the module name in usbipd.8

Fixes: 64e62426f40d ("staging: usbip: edit Kconfig and rename CONFIG options")
Acked-by: Shuah Khan <skhan@linuxfoundation.org>
Signed-off-by: Antonio Borneo <borneo.antonio@gmail.com>
Acked-by: matt mooney <mfm@muteddisk.com>
Link: https://lore.kernel.org/r/20200618000818.1048203-1-borneo.antonio@gmail.com
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 tools/usb/usbip/doc/usbipd.8 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/usb/usbip/doc/usbipd.8 b/tools/usb/usbip/doc/usbipd.8
index ac4635db3f03..fb62a756893b 100644
--- a/tools/usb/usbip/doc/usbipd.8
+++ b/tools/usb/usbip/doc/usbipd.8
@@ -73,7 +73,7 @@ USB/IP client can connect and use exported devices.
 
 .SH EXAMPLES
 
-    server:# modprobe usbip
+    server:# modprobe usbip-host
 
     server:# usbipd -D
         - Start usbip daemon.
-- 
2.16.4

